src/components/moderation/modlogs.ts

```diff
@@ -18,8 +18,8 @@ import { discord_timestamp } from "../../utils/discord.js";
 
 const moderations_per_page = 5;
 
-function is_autoremove(info: moderation_edit_info) {
-    return info.reason == "Auto" && info.moderator == WHEATLEY_ID;
```

Comment:
The intent of this function and the logic in `moderation_description` really is to check if it was an auto-removal. If the motivation for changing this is because of `WHEATLEY_ID`, I would recommend reworking so that there's a different way to get the bot ID. Alternatively I could do something to add an extra field to the moderation entries.

Reply:
I can take care of this later today

Reply:
I have pushed the change to rely on an `auto_removed` field

Reply:
nice, then I'll rebase onto that and move the Virustotal thing back